pandas/tests/indexes/test_numeric.py

```diff
@@ -17,6 +17,11 @@
 
 from pandas.tests.indexes.common import Base
 
+# For testing division by (or of) zero for Series with length 5, this
+# gives several scalar-zeros and length-5 vector-zeros
+zeros = tm.gen_zeros(5)
```

Comment:
just make this a fixture, then you don't need to add a parameterize on each function

Reply:
can you do this